deployments/operator/webhook/manifests.yaml

```diff
@@ -171,7 +171,7 @@ webhooks:
     service:
       name: webhook-service
       namespace: system
-      path: /pods-sgx
+      path: /mutate--v1-pod
```

Comment:
As discussed offline, this is seems funny but it fixing it doesn't seem to be an option (breaking change in the controller runtime).

Reply:
this is the way to go based on the CR maintainers so closing this review comment